lib/data/guides.json

```
@@ -37,10 +37,15 @@
     "url": "/guides/deploying-takeshape-with-vercel",
     "image": "https://og-image.now.sh/**Deploy%20a%20TakeShape%20Blog**%20with%20Next.js%20%26%20Vercel.png?theme=light&md=1&fontSize=100px&images=https%3A%2F%2Fassets.vercel.com%2Fimage%2Fupload%2Ffront%2Fassets%2Fdesign%2Fvercel-triangle-black.svg&images=https%3A%2F%2Fassets.vercel.com%2Fimage%2Fupload%2Fv1592998293%2Ffront%2Fguides%2Fogp%2Ftakeshape.svg",
     "name": "TakeShape",
+    "env": [
+      "NEXT_EXAMPLE_CMS_TAKESHAPE_API_KEY",
+      "NEXT_EXAMPLE_CMS_TAKESHAPE_PROJECT_ID",
+      "NEXT_EXAMPLE_CMS_TAKESHAPE_PREVIEW_SECRET"
```

`NEXT_EXAMPLE_CMS_` was removed:

```suggestion
      "TAKESHAPE_API_KEY",
      "TAKESHAPE_PROJECT_ID",
      "TAKESHAPE_PREVIEW_SECRET"
```